documentation/hacker-guide/source/topics/making-a-release.rst

```diff
@@ -0,0 +1,50 @@
+******************
+Release Check-list
+******************
+
+There are a number of manual steps involved in making an Open Dylan release and
+it's easy to forget one or two. Hopefully we can automate some of these but for
+now here is a manual check-list.
+
+#. Let people know you plan to make a release. Use the Gitter channel for this.
+   Because the Dylan ncommunity is small there's no need to make a release
+   branch; people can just be careful what they commit until the release is
+   out.
+
+#. Test on supported platforms.
+
+   * Do a 3-stage boostrap.
+   * Run test suites and try to figure out if anything serious is broken.
+     (This is an art, currently, and we should make all our tests pass.)
+   * Do some ad-hoc testing.  (Anything specific to mention here?)
+
+#. Update the version number in the sources
+
+   * In the release-info library
+   * In the build/packages/unix/README file
+   * Do a `git grep` for the previous release number, e.g., "2019.1" and see
+     if anything else needs to be updated.
+
+#. Update the release notes. Hopefully these have been maintained as changes
+   were made.  It may be worth scanning the commit logs.
+
+#. Tag the release
+
+   Use the format "vYYYY.n.m" where YYYY is the current year and n is the
+   release number during that year.  m is generally 0 but could be 1 if a quick
+   patch release were necessary. Note that these tags are used by the Dylan
+   package manager.
+
+#. Build the binaries for supported platforms
+
+#. Update other packages
+
+   * archlinux -- https://aur.archlinux.org/opendylan.git
+   * debian
```

Comment:
Is there a homebrew recipe too?

Reply:
I just added "homebrew" to the list for now.